### PR TITLE
TNO-787, 782, 785: Content form toolbar

### PR DESCRIPTION
**Pull request by jdtoombs**
Missing the first section that Alessia has worked on but will add that in once her code is merged

- moved actions to new content form toolbar
- alert section

image

### CODE REVIEW

File: app/editor/src/features/content/tool-bar/sections/form/AlertSection.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export const AlertSection: React.FC<IAlertSectionProps> = () => {
+  return (
+    <ToolBarSection
+      label="EMAIL ALERT"
+      icon={<FaPaperPlane />}
+      children={
+        <Row>
+          <ContentActions filter={(a) => a.name === ActionName.Alert} />
+          <div className="white-bg">not sent</div>
```
Comment (jdtoombs) [inline app/editor/src/features/content/tool-bar/sections/form/AlertSection.tsx:17]: I am just wondering what I should tie this to?

File: app/editor/src/features/content/tool-bar/ContentFormToolBar.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { ToolBar } from 'components/tool-bar/styled';
+import { ContentTypeName, IActionModel } from 'hooks';
+
+import { ActionSection, AlertSection } from './sections/form';
+
+// import { ActionSection } from './sections/form';
```
Comment (Fosol) [inline app/editor/src/features/content/tool-bar/ContentFormToolBar.tsx:6]: remove

File: app/editor/src/features/content/tool-bar/sections/form/AlertSection.tsx (hunk 1 above)
Comment (Fosol) [reply app/editor/src/features/content/tool-bar/sections/form/AlertSection.tsx:17]: Nothing presently.  We don't have alerts designed yet let alone built.

**Review by Fosol — APPROVED** [review]

**Comment by jdtoombs** [comment]
@Fosol, closing this one to upload a PR that incorporates this + new content form to avoid more difficult merge conflicts 😋

**Pull request closed without merging**